### PR TITLE
Feat: Added SQL code, Model, and REPO

### PR DESCRIPTION
**Pull request by Moibrahi7**

### CODE REVIEW

**Review by anthonydmays — CHANGES_REQUESTED** [review]
Please integrate with the `LibraryDbDataLoader` and add a test.

**Comment by anthonydmays** [comment]
> Please integrate with the `LibraryDbDataLoader` and add a test.

I just realized I forgot to call out adding a test for this in the assignment details. I won't dock credit if you don't do it, but you still should.

**Comment by Moibrahi7** [comment]
Is user suppose to go under patron

File: lesson_14/db/db_app/src/main/java/com/codedifferently/lesson14/models/LibraryUserModel.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package com.codedifferently.lesson14.models;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "guests")
```
Comment (anthonydmays) [inline lesson_14/db/db_app/src/main/java/com/codedifferently/lesson14/models/LibraryUserModel.java:8]: Should be `@Table(name = "library_users")`.

**Review by anthonydmays — CHANGES_REQUESTED** [review]

**Pull request closed without merging**